docs/scripting-ref.rst

```diff
@@ -2477,3 +2477,104 @@ arguments passed to splash, `username` and `password`.
         -- Usually, wait for the submit request to finish
         -- ...
     end
+
+
+.. _splash-select:
+
+
+splash:select
+-------------
+
+Select the first HTML element from DOM of the current web page that matches the specified CSS selector.
+
+**Signature:** ``element = splash:select(selector)``
+
+**Parameters:**
+
+* selector - valid CSS selector
+
+**Returns:** a :ref:`splash-element`.
+
+**Async:** no.
+
+Using :ref:`splash-select` you can get the element that matches your specified CSS selector like using
+`document.querySelector`_ in your browser. The returned element is a :ref:`splash-element` which has many useful
+methods and almost all methods and attributes that element has in JavaScript.
```

Comment:
What happens when an element can't be found - is this correct that nil is returned? I think it should be documented. Returning nil also means el = assert(splash:select("")) can be used.